src/PerformanceProfilingHttpEndpoints.jl

```diff
@@ -176,8 +207,12 @@ function __init__()
     precompile(cpu_profile_endpoint, (HTTP.Request,)) || error("precompilation of package functions is not supposed to fail")
     precompile(_do_cpu_profile, (Int,Float64,Float64,Bool)) || error("precompilation of package functions is not supposed to fail")
     precompile(allocations_profile_endpoint, (HTTP.Request,)) || error("precompilation of package functions is not supposed to fail")
+    precompile(allocations_start_endpoint, (HTTP.Request,)) || error("precompilation of package functions is not supposed to fail")
+    precompile(allocations_stop_endpoint, (HTTP.Request,)) || error("precompilation of package functions is not supposed to fail")
     if isdefined(Profile, :Allocs) && isdefined(PProf, :Allocs)
         precompile(_do_alloc_profile, (Float64,Float64,)) || error("precompilation of package functions is not supposed to fail")
+        precompile(_start_alloc_profile, (Float64,)) || error("precompilation of package functions is not supposed to fail")
+        precompile(_stop_alloc_profile, ()) || error("precompilation of package functions is not supposed to fail")
     end
 end
 
```

Comment:
Please add a test for the new endpoint.

Our testing is admittedly pretty limited, but at least exercising the endpoint would be good:
https://github.com/JuliaPerf/PerformanceProfilingHttpEndpoints.jl/blob/main/test/runtests.jl#L54